AliceAdventure/app/Editor/Pages/css/layout.css

```diff
@@ -88,8 +89,10 @@ canvas {
   /* width: 80% !important; */
   /* height: auto;
   margin: 0 auto; */
-  width: 1024px;
-  height: 576px;
+  width: 53.78vw;
+  max-width: 97.78vh;
```

Comment:
Can you add comments about how these numbers are added?

Reply:
I will add comments on the next pull request. Thanks.